tests/unit/test_collections_endpoint.py

```diff
@@ -0,0 +1,183 @@
+from datetime import datetime
```

Comment:
This test must move into /app/tests ! otherwise it won't be started by `make test` or `./manage.py test`

Reply:
strange, it worked before and I am sure, in older versions the path was correct. Perhaps directory structure changes due to new testrunner?
How shell I proceed? Move the file?

Reply:
I moved the file to the new directory